src/app/app-routing.module.ts

```diff
@@ -0,0 +1,23 @@
+import { AuthGuard } from './components/auth/auth.guard';
+import { SignupComponent } from './components/auth/signup/signup.component';
+import { FormBuilderComponent } from './components/form-builder/form-builder.component';
+import { LoginComponent } from './components/auth/login/login.component';
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import {Routes, RouterModule} from '@angular/router'
```

Comment:
import order, library imports first, app imports after them with one line separation
```suggestion
import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { Routes, RouterModule } from '@angular/router'

import { AuthGuard } from './components/auth/auth.guard';
import { SignupComponent } from './components/auth/signup/signup.component';
import { FormBuilderComponent } from './components/form-builder/form-builder.component';
import { LoginComponent } from './components/auth/login/login.component';
```